axelor-base/src/main/java/com/axelor/apps/base/service/printing/template/PrintingTemplateService.java

```diff
@@ -20,8 +20,13 @@
 
 import com.axelor.apps.base.AxelorException;
 import com.axelor.apps.base.db.PrintingTemplate;
+import com.axelor.rpc.Response;
 import java.util.List;
 
 public interface PrintingTemplateService {
-  List<PrintingTemplate> getPrintingTemplates(String modelName) throws AxelorException;
+  List<PrintingTemplate> getActivePrintingTemplates(String modelName) throws AxelorException;
+
+  void addPrintButton(String model, Response response);
+
+  Boolean hasActivePrintingTemplates(String modelName);
```

Comment:
this method is only called from `addPrintButton`, so please make it `protected`

Reply:
Keeping it public as it will be referenced from new service class `PrintingTemplateMetaService`